hydrus/conf.py

```diff
@@ -0,0 +1,61 @@
+"""
+Global variables are loaded or set here:
+    DEBUG
+    PORT
+    API_NAME
+    DB_URL
+    APIDOC_OBJ
+    HYDRUS_SERVER_URL
+"""
+import os
+import logging
+
+from os.path import abspath, dirname
+from pathlib import Path
+
+from importlib.machinery import SourceFileLoader
+
+logger = logging.getLogger(__file__)
+
+try:
+    DEBUG = bool(os.environ['DEBUG'])
+except KeyError:
+    DEBUG = False
+
+# load form environment (as many globals as possible shall be in
+# environment configuration)
+try:
+    PORT = int(os.environ['PORT'])
+    API_NAME = os.environ['API_NAME']
+    DB_URL = os.environ['DB_URL']
```

Comment:
Don't you think there should be some default values in case env variables are not set or the user tries to run app.py without docker?

Reply:
these ones have to be set or the user has not control over the server.

I added the test variables in `travis.yml`